net/Socket.hpp

```diff
@@ -398,12 +412,19 @@ class Socket
         , _open(_fd >= 0)
         , _creationTime(creationTime)
         , _lastSeenTime(_creationTime)
+        , _bytesSent(0)
```

Comment:
It seems that the change to move _bytesSent and _bytesRcvd into the base class is effectively only so they are available for Socket::streamStats? So I wonder if that's something we need to do?

Reply:
For stats and offering it as an criteria for checkTimeout / checkRemoval (used in the latter).
When I unified it in the Socket, idea was to push down general properties (like open) etc.

Reply:
reordered the specific commit .. resulting in visible usage and compile clean commits